### DISCUSSION TITLE
Long term Timelapse issue

### DISCUSSION THREAD

**Discussion by Jeando17300** [gopro/labs#107] [Q&A] [answered]
Hello ! I’ve a GoPro 10 black
I want to do long term Timelapse. Hello t works fine as long as I’m using the GoPro battery… I bought the usb adapter to be able to use a powerbank battery… but 2 issues :
If GoPro battery not inside the camera give me an error message that there is no 2A input (which is wrong because this is a 2A output powerbank AND I can use the GoPro if powered manually) AND if I put the GoPro battery inside with the powerbank the GoPro constantly go on and off after launching the Timelapse script…
So it is not working… can anyone help me on this ?
Thanks !

**Comment by dnewman-gpsw** [comment, marked as answer]
Some power banks don't correctly report their power rating.  There is a Labs hack for this, that forces the camera to assume 2A for sources that don't report. ![image](https://user-images.githubusercontent.com/10856084/194360888-793a37b6-6794-4136-a26c-8642d1fa2a74.png)

**Reply by Jeando17300** [reply]
Thanks I'm going to try this !

**Comment by Jeando17300** [comment]
Great it works fine ! Thank you ! Do you know if  only have to do this only one time, or maybe after each new firmware ? 
Have a good day.

**Reply by dnewman-gpsw** [reply]
Once per camera.  It saves permanently, only erased with this Labs command: !RESET

**Reply by Jeando17300** [reply]
Great 👍
